configs/immy.json

```
@@ -1,12 +1,6 @@
 {
   "index_name": "immy",
-  "start_urls": [
-    "https://docs.immy.bot/#/immy-commands",
-    "https://docs.immy.bot/#/scripts"
-  ],
-  "stop_urls": [
-    "\\?id="
-  ],
+  "sitemap_urls": ["https://docs.immy.bot/sitemap.xml"],
```

```suggestion
  "sitemap_urls": ["https://docs.immy.bot/sitemap.xml"],
  "start_urls": [
    "https://docs.immy.bot/"	
  ],	
```

Also, make sure to update your **sitemap.xml** because the domain is **localhost:8081** for now